docs/basics/gas-fees.md

```diff
@@ -1,11 +1,12 @@
 <!--
 order: 4
-synopsis: This document describes the default strategies to handle gas and fees within a Cosmos SDK application.
 -->
 
 # Gas and Fees 
 
-## Pre-requisite Readings {hide}
+This document describes the default strategies to handle gas and fees within a Cosmos SDK application. {synopsis}
+
+### Pre-requisite Readings
```

Comment:
`{hide}` hides the block, unless it's the "Pre-requisite Readings", which is shown no matter what (on purpose).

I've removed `{hide}` from all "Pre-requisite Readings" to avoid confusion.